cmd/git-sync/main.go

```diff
@@ -397,6 +401,20 @@ func main() {
 		handleConfigError(log, true, "ERROR: --period must be at least 10ms")
 	}
 
+	var syncSig syscall.Signal
+	if *flSyncOnSignal != "" {
+		if num, err := strconv.ParseInt(*flSyncOnSignal, 0, 0); err == nil {
+			// sync-on-signal value is a number
+			syncSig = syscall.Signal(num)
+		} else {
+			// sync-on-signal value is a name
+			syncSig = unix.SignalNum(*flSyncOnSignal)
+		}
+		if unix.SignalName(syncSig) == "" {
+			handleConfigError(log, true, "ERROR: --sync-on-signal unknown signal")
```

Comment:
nit: ...must be a valid signal name or number

Reply:
No, that's fine, now that you point it out ... it's obviously wrong.